core/src/main/java/org/mobilitydata/gtfsvalidator/input/GtfsZipFileInput.java

```diff
@@ -32,28 +33,43 @@
 public class GtfsZipFileInput extends GtfsInput {
   private final ImmutableSet<String> filenames;
   private final ZipFile zipFile;
+  private boolean isMacZip;
 
-  public GtfsZipFileInput(ZipFile zipFile) {
+  public GtfsZipFileInput(ZipFile zipFile, String zipFileName) {
     this.zipFile = zipFile;
-
+    StringBuilder strBuilder = new StringBuilder();
```

Comment:
Using StringBuilder is an overkill for that. you should use se a simple string concatenation instead:
```
String macDirectory = zipFileName + "/";
```

Reply:
Fine. 
I just read that the java compiler does a StringBuilder under the hood when you use +. So the two forms are equivalent. 
I just feel that 
```
String macDirectory = zipFileName + "/";
```
Is more readable.
But that's a question of style.
I learn something new, that's good.